processing/src/main/java/org/apache/druid/query/aggregation/ExpressionLambdaAggregator.java

```diff
@@ -29,12 +29,19 @@
   private final Expr lambda;
   private final ExpressionLambdaAggregatorInputBindings bindings;
   private final int maxSizeBytes;
+  private boolean uninitializedNullValue;
 
-  public ExpressionLambdaAggregator(Expr lambda, ExpressionLambdaAggregatorInputBindings bindings, int maxSizeBytes)
+  public ExpressionLambdaAggregator(
+      final Expr lambda,
+      final ExpressionLambdaAggregatorInputBindings bindings,
+      final boolean initiallyNull,
+      final int maxSizeBytes
+  )
   {
     this.lambda = lambda;
     this.bindings = bindings;
     this.maxSizeBytes = maxSizeBytes;
+    this.uninitializedNullValue = initiallyNull;
```

Comment:
I would suggest to name this as `useNullIfUninitialized` for clarity - the current name didn't seem like a boolean question to me. I would also prefer any other better name.